sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1247,9 +1201,7 @@ object functions {
    * @since 1.5.0
    */
   @scala.annotation.varargs
-  def least(columnName: String, columnNames: String*): Column = if (columnNames.isEmpty) {
-    sys.error("LEAST takes at least 2 parameters")
-  } else {
+  def least(columnName: String, columnNames: String*): Column = {
```

Comment:
Nit: Should we remove the `columnName` version API for both `least` and `greatest`?